src/main/java/racingcar/CarInfo.java

```diff
@@ -0,0 +1,23 @@
+package racingcar;
+
+import lombok.Data;
+
+@Data
+public class CarInfo {
+    private int prevDistance = 0;
+    private int currentDistance = 0;
+    private int resultOfRand;
+
+    public void moveFoward() {
+        prevDistance = currentDistance;
+        currentDistance = prevDistance + 1;
+    }
+
+    public boolean isMoved() {
+        return currentDistance - prevDistance > 0;
```

Comment:
```suggestion
    public boolean isMoved(value) {
        return value > 0;
```

`isMoved`는  사용하는 입장에서`비교 검증`의 책임이 가장 큰것 같아요. 
이미 계산된 (`currentDistance - prevDistance`) 의 값을 넣어준다라는 관점으로 접근한다면, 메소드의 책임이 좀 더 명확해지지 않을까요?


물론, 현재 로직에서는 큰문제는 없습니다. 
이번 과정을 진행하시면서 메소드의 책임을 분명히 하고 하나의 개념만 작업을 할 수있도록 훈련해보시면 좋을것 같아요!